app/controllers/solidus_square/callback_actions_controller.rb

```diff
@@ -11,21 +11,31 @@ def square_checkout
         access_token: SolidusSquare.config.square_access_token,
         environment: SolidusSquare.config.square_environment,
         location_id: SolidusSquare.config.square_location_id
-      ).checkout(order, redirect_url)[:checkout_page_url]
+      ).checkout(current_order, redirect_url)[:checkout_page_url]
```

Comment:
Why do you change this? 🤔

Reply:
https://github.com/nebulab/solidus_square/pull/37/commits/917128edce435cd2fb5c75976678be8f8a5881cf